packages/web/src/client/components/Headers/ProjectHeader.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import { ProjecHeader } from './ProjectHeader'
+
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import { StoryFixtures } from '@types'
+
+export default {
+  title: 'Headers/Project',
+  component: ProjecHeader,
+} as ComponentMeta<typeof ProjecHeader>
+
+export const Header: ComponentStory<typeof ProjecHeader> = () => (
+  <ProjecHeader
+    title={fixtures.headerContents.title}
```

Comment:
this should be using the same prop spreading as we're using elsewhere, uniformity 👍🏼